readthedocs/integrations/utils.py

```diff
@@ -23,3 +28,16 @@ def normalize_request_payload(request):
         except AttributeError:
             pass
     return request_payload
+
+
+def get_secret(size=64):
+    """
+    Get a random string of `size` bytes.
+
+    :param size: Number of bytes
+    """
+    secret = os.urandom(size)
+    if six.PY2:
```

Comment:
Do we still need this?

Reply:
No.

This PR should be probably updated by removing all the py2 compatible code.

Reply:
Besides removing this, we should execute `pyupgrade` over these new files, remove all the `six` and `future` references, etc.

Maybe it's good to follow #4543 checklist